### ISSUE TITLE
NTEE coverage

### ISSUE THREAD

**Issue by hampelm** [detroitledger/gnl_profile#148]
Some orgs in our database have multiple NTEE codes in our database -- say, Matrix Human Services: https://www.detroitledger.org/#!/organizations/331

However, it looks like there is no NTEE data in the BMF for them:
https://data.detroitledger.org:6001/ein/381358015

There also are lots of orgs where we have no NTEE(s). 

Turns out the answer why is longer than I wanted -- Jessica, I imagine you went through this maze for the custom work we did. 

I found this thread on guidestar -- particularly this answer: https://community.guidestar.org/thread/2194

> Our search engine uses the NTEE (National Taxonomy of Exempt Entities) Classification System developed by the National Center for Charitable Statistics as part of its keyword searching criteria.  In the past the IRS assigned the initial code for organizations. However this is no longer the case. 

[From 2009](http://nccsweb.urban.org/knowledgebase/detail.php?linkID=922&category=120&xrefID=3655):

> Sixty-one percent of organizations in our files do NOT have an IRS-assigned NTEE code. Approximately 26% have a code that matches the IRS code at the 3-character level.

http://nccsweb.urban.org/knowledgebase/detail.php?linkID=728: 

> All of the more than 2.1 million organizations in the NCCS databases are classified in NCCS's master list of NTEE codes. These codes come from a variety of different sources.
> 
> Since the late 1990s, the IRS has classified new organizations as they applied for exemption. As of 2006, more than 320,000 organizations have been classified by the IRS "determination specialists,

Sounds like we may be able to get some data from NCCS, but it may cost: http://nccs.urban.org/database/index.cfm

Right now, in our Michigan BMF extract, only 1/2 the orgs have a single NTEE.

@jessicamcinchak: Would you be up for looking into the best way(s) to make our NTEE coverage as comprehensive as possible?

**Comment by jessicamcinchak** [comment]
yep! will have time this week to do some research and make a plan. more soon!

**Comment by jessicamcinchak** [comment]
Some findings and ideas brewing in this gdoc: https://docs.google.com/document/d/1qtKBuDq8vJFsA7GNOqqfEWTwP90c5ptP3nMfK4BOols/edit

Basically, once we import new orgs via BMF, I think we'll be able to fill in most missing codes using Guidestar and [an existing NCCS search tool](http://nccsweb.urban.org/PubApps/search.php) (no fee) - both of which offer additional NTEE classification methods/assignments on top of BMF baseline codes (assigned only at point of registering for tax-exempt status via IRS Form 1023 or 1024).

Still learning specifics of NTEE requirements to better understand total coverage we should be aiming for.
